namespace: adding a dedicated entry for operator namespace in common (#13080)
rook/rook: Documentation/Storage-Configuration/Advanced/ceph-configuration.md

forget to set `ROOK_CURRENT_NAMESPACE_ONLY`), or you can leave it at the same value for every
Ceph cluster if you only wish to have one Operator manage all Ceph clusters.

!!! note

>     The namespace `ROOK_OPERATOR_NAMESPACE` must be created manually if it does not exist:

Reviewer: If they are configuring a different namespace, but the operator and cluster namespace are the same, this is not needed. How about this? And to keep the note simple, IMO we don't need to give them the example command to create the namespace.
```suggestion
    If the operator namespace is different from the cluster namespace, the operator namespace must be created before running the steps below. (The cluster namespace does not need to be created first, as it will be created by `common.yaml` in the script below.)
```

forget to set `ROOK_CURRENT_NAMESPACE_ONLY`), or you can leave it at the same value for every
Ceph cluster if you only wish to have one Operator manage all Ceph clusters.

> !!! note

Reviewer: I'm thinking the flow will be more natural if this is just a paragraph instead of a separate note. With the note, it seems awkward that the console command is separate, but it should belong to the note. The rendering would make the console command appear separate from the note.
```suggestion
```

    If the operator namespace is different from the cluster namespace, the operator namespace must be

> 	created before running the steps below. (The cluster namespace does not need to be created first,

Reviewer: I think my suggestion to use parenthesis was not good, it's better to not use them. 
```suggestion
	created before running the steps below. The cluster namespace does not need to be created first,
```

> 	as it will be created by `common.yaml` in the script below.)

Reviewer: The docs rendering is a bit sensitive. A blank line is usually needed before and after console blocks. 
```suggestion
	as it will be created by `common.yaml` in the script below.
	
```
